homeassistant/components/solax/sensor.py

```diff
@@ -27,24 +29,32 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.event import async_track_time_interval
 
-DEFAULT_PORT = 80
+from .const import DOMAIN
 
+DEFAULT_PORT = 80
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    cv.deprecated(DOMAIN),
```

Comment:
Sorry, it seems that cv. deprecated is not working for PLATFORM_SCHEMA.
https://github.com/home-assistant/core/pull/56869#discussion_r719954461

Reply:
Removed